lib/ignored-paths.js

```diff
@@ -169,6 +158,29 @@ class IgnoredPaths {
     }
 
     /**
+     * read ignore filepath
+     * @param {string} filePath, file to add to ig
+     * @returns {array} raw ignore rules
+     */
+    readIgnoreFile(filePath) {
+        if (!this.cache[filePath]) {
```

Comment:
If an `.eslintignore` file exists and happens to be empty, the cache won't get used because `this.cache[filePath]` will be the empty string.

One way to resolve this would be to use `if (typeof this.cache[filePath] === "undefined")` instead.